curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646b0582494fbe03ab07971c.md

```diff
@@ -0,0 +1,85 @@
+---
+id: 646b0582494fbe03ab07971c
+title: Step 13
+challengeType: 0
+dashedName: step-13
+---
+
+# --description--
+
+You should target the class named `date-svg` and set the `width` to `30px` and `height` to `30px`.
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed